helios-services/src/test/java/com/spotify/helios/agent/TaskRunnerTest.java

```diff
@@ -58,6 +69,11 @@
   @Mock private Clock clock;
   @Mock private ContainerDecorator containerDecorator;
 
+  @Mock private TaskRunner.NopListener mockListener;
```

Comment:
I guess this is a matter of opinion on style, but.... If these mocks are only used in a single test consider declaring them locally in that test.

E.g. instead of having members just do `final ImageInfo mockImageInfo = mock(ImageInfo.class)` etc in the test method.